Simulator/Vertices/NG911/All911Vertices.h

```diff
@@ -120,6 +120,11 @@ class All911Vertices : public AllVertices {
    /// endStep (exclusive)
    virtual void loadEpochInputs(uint64_t currentStep, uint64_t endStep) override;
 
+   /// unused virtual function placeholder
+   virtual void registerHistoryVariables() override
+   {
+   }
+
```

Comment:
Do we have an issue to get 911 working with the new Recorders?

Reply:
Yes, we do #574 : Enhance Recordable Class to Support Variables for NG911 Simulation